src/Fantomas.Core.Tests/ShebangTests.fs

```diff
@@ -0,0 +1,44 @@
+module Fantomas.Core.Tests.ShebangTests
+
+open NUnit.Framework
+open FsUnit
+open Fantomas.Core.Tests.TestHelper
+
+[<Test>]
+let ``should keep a shebang, 2367`` () =
+    let codeSnippet =
+        """#!/usr/bin/env -S dotnet fsi
+// random licensing stuff
+
+open System
+
+printfn "the best thing we've ever done"
+"""
+
+    formatSourceString false codeSnippet config |> should equal codeSnippet
```

Comment:
Please use the typical test template, https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#pull-request-ground-rules.

Even if the before and after are the same, all our (2000+) other tests look like this. There is no need to diverge from this.

Reply:
Done!